central/complianceoperator/v2/report/manager/complianceReportgenerator/consts.go

```diff
@@ -0,0 +1,13 @@
+package complianceReportgenerator
+
+const (
+	defaultEmailBodyTemplate = "{{.BrandedPrefix}} has identified {{.ComplianceStatus}} profile checks for clusters scanned by your \n" +
+		"schedule configuration parameters. The attached report lists those checks and associated details to aid with remediation. \n" +
+		"Profiles:{{.Profile}} \n" +
+		"Passing:{{.Pass}} checks \n" +
+		"Failing:{{.Fail}} checks \n" +
+		"Mixed:{{.Mixed}} checks \n" +
+		"Clusters {{.Clusters}} scanned"
+
+	defaultSubjectTemplate = "{{.BrandedPrefix}} Compliance Report For {{.ScanConfig}} Profiles {{.Profiles}}"
```

Comment:
We could have a really large number of profiles in a scan config - will this still work? Any subject length restrictions? Special characters in profile names?

How about we say for 10 profiles?